engine/graphics/src/graphics.cpp

```diff
@@ -1611,6 +1611,11 @@ namespace dmGraphics
     }
     bool IsContextFeatureSupported(HContext context, ContextFeature feature)
     {
+        if (CONTEXT_FEATURE_VSYNC == feature)
+        {
+            AdapterFamily family = GetInstalledAdapterFamily();
+            return !(family == ADAPTER_FAMILY_NULL || family == ADAPTER_FAMILY_NONE);
+        }
```

Comment:
Minimal change in order to avoid adding such code in all backends (i.e. consoles too)